tests/test_acasclient.py

```diff
@@ -1797,4 +1595,299 @@ def test_044_author_and_role_apis(self):
+            },
+            {
+                "errorLevel": "warning",
+                "message": "Experiment '4 parameter D-R - 2018-05-08' already exists, so the loader will delete its current data and replace it with your new upload. If you do not intend to delete and reload data, enter a new Experiment Name."
+            }
+        ]
+        self.assertCountEqual(response['errorMessages'], expected_messages)
+        for message in response['errorMessages']:
+            # This matches the response error message and level to the expected message and level
+            expectedResult = [m for m in expected_messages if m['errorLevel'] == message['errorLevel'] and m['message'] == message['message']]
+            # Should return 1 and only 1 match
+            self.assertEqual(len(expectedResult), 1)
+
+    def test_dose_response_experiment_loader(self):
+        """Test dose response experiment loader."""
+        data_file_to_upload = Path(__file__).resolve()\
+            .parent.joinpath('test_acasclient', '4 parameter D-R.csv')
+        request = {
+            "data_file": data_file_to_upload,
+            "user": "bob",
+            "dry_run": True,
+            "model_fit_type": "4 parameter D-R",
+            "fit_settings": {
+                "smartMode":True,
+                "inactiveThresholdMode":True,
+                "inactiveThreshold":20,
+                "theoreticalMaxMode":False,
+                "theoreticalMax":None,
+                "inverseAgonistMode":False,
+                "max":{
+                    "limitType":"none"
+                },
+                "min":{
+                    "limitType":"none"
+                },
+                "slope":{
+                    "limitType":"none"
+                },
+                "baseline":{
+                    "value":0
+                }
+            }
+        }
+        response = self.client.\
+            dose_response_experiment_loader(**request)
+        self.assertIn("experiment_loader_response", response)
+        self.assertIn('results', response["experiment_loader_response"])
+        self.assertIn('errorMessages', response["experiment_loader_response"])
+        self.assertIn('hasError', response["experiment_loader_response"])
+        self.assertIn('hasWarning', response["experiment_loader_response"])
+        self.assertIn('transactionId', response["experiment_loader_response"])
+        self.assertIsNone(response["experiment_loader_response"]['transactionId'])
+        self.assertIsNone(response['dose_response_fit_response'])
+
+        # When loading dose response data and doing a curve fit we shold NOT get a dose response summary table because acas isn't evaluating the uploaded
+        # curve fit parameters (it replaces it with it's own curve fits)
+        self.assertTrue(response["experiment_loader_response"]['results']['htmlSummary'].find("bv_doseResponseSummaryTable") == -1)
```

Comment:
This was an existing overall test but added this assert to make sure we don't get a dose response summary table when just fitting curves.